src/patterns/index.md.njk

```diff
@@ -9,3 +9,8 @@ tasks and scenarios.
 Some of the patterns in this section contain coded examples, others are written
 guidance. Patterns often use single or combinations of components and describe
 their use in context of their relationship to the user.
+
+On the most part, if you are using the [GOV.UK Prototype Kit](https://govuk-prototype-kit-beta.herokuapp.com) or
```

Comment:
Should this be 'for the most part'? Has @amyhupe looked at these?

Reply:
@joelanman Good shout, this is with Amy now